app/config_objects/district_config.rb

```diff
@@ -0,0 +1,69 @@
+class DistrictConfig
+
+  cattr_accessor :remote_filenames
```

Comment:
Here's a style suggestion, that you might be into or not.  No need to hold up shipping this, and up to you whether this seems worthwhile or not.

This is written with only class methods, which makes it hard to see what's a public method and what's not. 
 It also keeps state, so the order you call these in matters, but the method names describe what they do, not how they should be used.

One option might be to write this as a plain class, and then expose only one class method `remote_filenames`. 
 That would read the yaml file and everything is stateless, or there could be a second class method like `load!` or something that has to be called in the Rails initializer and `remote_filenames` raises if not.

Reply:
I like the stateless idea, that reduces some complexity because it lets us get out of the whole initialization game. Let me work on that and add some more commits to this PR.